src/test/java/ir/sahab/dockercomposer/DockerComposeRunnerTest.java

```diff
@@ -3,18 +3,46 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import org.assertj.core.api.Condition;
+import org.junit.Assert;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.zeroturnaround.exec.ProcessExecutor;
 
+import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Map;
+import java.util.concurrent.TimeoutException;
 
 public class DockerComposeRunnerTest {
 
     private static final int PORT = 2181;
     private static final Condition<Service> accessible = new ServiceAccessible(PORT);
 
+
+    @ClassRule
+    public static DockerCompose dockerCompose = DockerCompose.builder()
```

Comment:
Why did u added another docker dependency? Couldn't test u scenario with existing docker image? (zookeeper)

Reply:
Because I want to inject my user  id and jplock/zookeeper:3.4.10 image just run with user id 0 (or root).